packages/components/psammead-media-player/src/Placeholder/index.jsx

```diff
@@ -12,15 +12,15 @@ const StyledPlaceholder = styled.div`
   height: 100%;
 `;
 
-const Placeholder = ({ onClick, src }) => (
+const Placeholder = ({ onClick, srcset }) => (
   <StyledPlaceholder onClick={onClick}>
-    <Image alt="Image Alt" src={src} />
+    <Image alt="Image Alt" srcset={srcset} />
```

Comment:
The documentation for `Image` suggests `src` is a required prop so I think this change will violate that? https://github.com/bbc/psammead/tree/latest/packages/components/psammead-image

Also, I'd suggest this component should follow a similar approach and support either src or src + srcset like the image component does.